apps/logging/serializers.py

```diff
@@ -117,6 +117,15 @@ class FHIRRequest(Request):
     def uuid(self):
         return self.req.headers.get('BlueButton-OriginalQueryId')
 
+    def fhir_id(self):
+        # BB2-135 add event field 'fhir_id' per requirement of BFD Insights
+        fhir_id = ''
+        bid = self.req.headers.get('BlueButton-BeneficiaryId')
+        fhir_id = str(bid) if bid is not None else ''
```

Comment:
@JFU-GIT This code could be replaced with:
```python
        fhir_id = self.req.headers.get('BlueButton-BeneficiaryId', '')
```

Reply:
that's concise, thanks!